pkg/webhook/inject/setup_test.go

```diff
@@ -21,6 +21,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/chaos-mesh/chaos-mesh/pkg/log"
```

Comment:
we need an alias for this import because the symbol `log` is already used under this package.